package.json

```diff
@@ -3,9 +3,16 @@
   "version": "0.5.3",
   "description": "FSA-compliant promise middleware for Redux.",
   "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
   "scripts": {
-    "test": "make test",
-    "prepublish": "make clean build"
+    "build": "babel src --out-dir lib",
+    "precommit": "pretty-quick --staged",
+    "lint": "xo",
+    "test": "jest",
+    "posttest": "npm run lint",
```

Comment:
Do we need this? We don't have this in the other projects.

Reply:
Removed